app/components/arclight/document_component.html.erb

```diff
@@ -31,12 +31,7 @@
 </div>
 
 <%= render Arclight::MetadataSectionComponent.with_collection(component_metadata_partials, metadata_attr: { layout: Arclight::UpperMetadataLayoutComponent }, presenter: presenter, classes: ['al-metadata-section']) unless document.collection? %>
-
-<div class='row'>
-  <div class='col-md-12'>
-    <%= embed %>
-  </div>
-</div>
+<%= embed %>
```

Comment:
i think this needs 1rem of margin to separate it from the metadata section below

Reply:
I added this in the EmbedComponent